### DISCUSSION TITLE
官网钉钉群过期了，请问能更新一下吗

### DISCUSSION THREAD

**Discussion by resetsix** [antvis/X6#4432] [General]
官网的钉钉群邀请过期很久了。

https://x6.antv.antgroup.com/tutorial/about#%EF%B8%8F-%E5%A6%82%E4%BD%95%E4%BA%A4%E6%B5%81

**Comment by resetsix** [comment]
@zlecheng你好，请问还有钉钉群吗

**Reply by zlecheng** [reply]
> @zlecheng你好，请问还有钉钉群吗

![image](https://github.com/user-attachments/assets/031862d1-6740-4e4d-a42c-8de6d2d469a2)
